libs/auth-api-lib/src/lib/delegations/delegations-index.service.ts

```diff
@@ -250,13 +262,38 @@ export class DelegationsIndexService {
   ) {
     const delegations = await this.delegationsIncomingRepresentativeService
       .findAllIncoming({ nationalId }, useMaster)
-      .then((d) => d.map(toDelegationIndexInfo))
+      .then((d) => {
+        // append the personal representative right type code to the delegation type in index
+        const delegationsWithRights = d.reduce(
+          (acc: DelegationDTOWithStringType[], delegation) => {
+            if (!delegation.rights) {
+              return acc
+            }
+
+            const delegations = delegation.rights.map((right) => ({
+              ...delegation,
+              type: `${delegation.type}:${right.code}`,
+              rights: [right],
```

Comment:
Shouldn´t this be mapping to `DelegatoinDTOWithStringType`?
That is based on the `DelegationDto` which doesn't include the `rights` property. So maybe use `as` to force the type and remove the `rights`.